pkg/slo-controller/nodemetric/nodemetric_controller.go

```diff
@@ -18,23 +18,30 @@ package nodemetric
 
 import (
 	"context"
+	"fmt"
+	"reflect"
 
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/klog/v2"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	slov1alpha1 "github.com/koordinator-sh/koordinator/apis/slo/v1alpha1"
+	"github.com/koordinator-sh/koordinator/pkg/slo-controller/config"
+	"github.com/koordinator-sh/koordinator/pkg/slo-controller/noderesource"
 )
 
 // NodeMetricReconciler reconciles a NodeMetric object
 type NodeMetricReconciler struct {
 	client.Client
 	Scheme *runtime.Scheme
+	config noderesource.Config
```

Comment:
add a TODO here. it's weird that nodemetric import config from noderesource

Reply:
done